services/volume-inspector/api/api_test.go

```diff
@@ -0,0 +1,37 @@
+package api_test
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+	"github.com/eudat-gef/gef/services/volume-inspector/api"
+)
+
+var (
+	server   *httptest.Server
+	reader   io.Reader
+	lsUrl string
+)
+
+func init() {
```

Comment:
It's recommended to use `func TestMain(m *testing.M)` for setup/teardown when testing. Init should be used for initialization of global variables. See https://golang.org/pkg/testing/